index.html

```diff
@@ -1,12 +1,149 @@
 	<title>Bitcoin-Conference 2021 </title>
 </head>
+
 <body>
-	
+	<!-- Header section of the mobile website version starts here -->
+	<header>
+		<nav>
+			<button type="button" class="btn hambuger-menu"><svg xmlns="http://www.w3.org/2000/svg" width="30"
+					height="30" fill="currentColor" viewBox="0 0 24 24" stroke="currentColor">
+					<path stroke-linecap="round" stroke-linejoin="round" stroke-width="3" d="M4 6h16M4 12h16M4 18h16" />
+				</svg></button>
+		</nav>
+		<section class="main-wrapper">
+			<div class="main-nav">
+				<ul id="main-bar">
+					<li><i class="fab fa-facebook-f"></i></li>
+					<li><i class="fab fa-twitter"></i></li>
+					<li><a href="#">Logout</a></li>
+					<li><a href="#">Mypage</a></li>
+					<li><a href="#">English</a></li>
+				</ul>
+			</div>
+			<div class="program-section">
+				<button class="nav__close" id="close-button" type="button">
+					<svg xmlns="http://www.w3.org/2000/svg" width="40" height="40" fill="currentColor"
+						viewBox="0 0 24 24" stroke="currentColor">
+						<path stroke-linecap="round" stroke-linejoin="round" stroke-width="2"
+							d="M6 18L18 6M6 6l12 12" />
+					</svg>
+				</button>
+				<a href="index.html"> <img src="Assets/Icons/running.jpeg" id="logo" alt="logo"></a>
+				<ul class="nav-menu">
+					<li class="about"><a href="about.html">About</a></li>
+					<li><a href="#">Program</a></li>
+					<li><a href="#">Join</a></li>
+					<li><a href="#">Sponsors</a></li>
+					<li><a href="#">News</a></li>
+					<li id="camp"><a href="#">Campaign</a></li>
+				</ul>
+			</div>
+		</section>
+		<section class="home-page">
+			<p class="moto">Keeping You Healthy</p>
+			<h1 class="main-header">
+				RAY'S ELEMENT AQUA <br /> GLOBAL SUMMIT <br /> 2021 </h1>
+			<p class="main-text">Aqua's Gym is the Number one training facility in the west-side, A joyfull arena
+				where
+				we make you meet your fitness Goals happening yearly in over 30 states.</p>
+			<time datetime="2021-08-20"> 2021.08.20(FRI) &sim; 26(THU) </time>
+			<blockquote>&commat; Mana Convention Center Wynwood Miami</blockquote>
+		</section>
+	</header>
+	<section class="main-program">
+		<h2>Main Program</h2>
+		<div class="event-container">
+			<div class="main-event">
+				<img src="Assets/Icons/lecture.png" alt="lecture-flaticon">
+				<h3>Lecture</h3>
+				<p class="lectures">Listen to top fitness coaches and experts as they share their knowledge on how to
+					live a healthy
+					lifestle.</p>
+			</div>
+			<div class="main-event">
+				<img src="Assets/Icons/exhibiiton.png" alt="exhibition-flaticon">
+				<h3>Exhibition</h3>
+				<p class="workshop">Take a look around and get inspired by the different ways you can keep yourself fit
+					and healthy.
+				</p>
+			</div>
+			<div class="main-event">
+				<img src="Assets/Icons/forum.png" alt="forum-flaticon">
+				<h3>Forum</h3>
+				<p class="forum">Have the opportunity to share your thoughts and opinions with experts on topics that
```

Comment:
* I suggest you use `article ` tags for an article as it is the right semantic tag which helps improve browser readabilty